cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -601,9 +602,248 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
+              |mine.private.exampleProcedure
+              |mine.private.exampleProcedure1
+              |mine.private.exampleProcedure2
+              |your.exampleProcedure""".stripMargin)
+
+          query("GRANT EXECUTE PROCEDURE * ON DBMS TO procedureGlobbing1", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureGlobbing1' can then run procedures all the procedures.")
+            statsOnlyResultTable()
+          }
+
+          query("GRANT EXECUTE PROCEDURE mine.*.exampleProcedure ON DBMS TO procedureGlobbing2", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureGlobbing2' can then run procedures `mine.public.exampleProcedure` and `mine.private.exampleProcedure`, but none of the others.")
+            statsOnlyResultTable()
+          }
+
+          query("GRANT EXECUTE PROCEDURE mine.*.exampleProcedure? ON DBMS TO procedureGlobbing3", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureGlobbing3' can then run procedures `mine.public.exampleProcedure1`, `mine.private.exampleProcedure1` and `mine.private.exampleProcedure2`, but none of the others.")
+            statsOnlyResultTable()
+          }
+
+          query("GRANT EXECUTE PROCEDURE *.exampleProcedure ON DBMS TO procedureGlobbing4", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureGlobbing4' can then run procedures `your.exampleProcedure`, `mine.public.exampleProcedure` and `mine.private.exampleProcedure`, but none of the others.")
+            statsOnlyResultTable()
+          }
+
+          query("GRANT EXECUTE PROCEDURE mine.public.exampleProcedure* ON DBMS TO procedureGlobbing5", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureGlobbing5' can then run procedures `mine.public.exampleProcedure`, `mine.public.exampleProcedure1` and `mine.public.exampleProcedure42`, but none of the others.")
+            statsOnlyResultTable()
+          }
+        }
+      }
+
       section("Granting `ALL DBMS PRIVILEGES`", "administration-security-administration-dbms-privileges-all", "enterprise-edition") {
         p(
-          """The right to create, drop, assign, remove and show roles, create, alter, drop and show users, create and drop databases and show, assign and remove privileges can be achieved with a single command:""".stripMargin)
+          """The right to create, drop, assign, remove and show roles, create, alter, drop and show users, create and drop databases,
+            |show, assign and remove privileges and execute all procedures with elevated privileges can be achieved with a single command:""".stripMargin)
```

Comment:
This list is a bit confusing, perhaps a bulleted list would make this easier to read? Something like:

> The right to perform the following privileges can be achieved with a single command:
> 
> - create roles
> - drop roles 
> - assign roles
> - remove roles 
> - show roles
> - create users
> - alter users
> - drop users
> - show users
> - create databases
> - drop databases
> - show privileges
> - assign privileges
> - remove privileges
> - execute all procedures with elevated privileges